src/pages/docs/tutorials/tutorial-typescript-enact/adding-a-new-component/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Add a New Component Using TypeScript
+github: https://github.com/enactjs/docs/blob/develop/src/pages/docs/tutorials/tutorial-typescript-basic/app-setup/index.md
+order: 3
+---
+
+### Creating a Counter Component
+
+- Create a **Counter** folder in **src/components** and add a **Counter.tsx** file
+
+```none
+./src/components/Counter/Counter.tsx
+```
+
+### Counter using Typescript
+
+We can now populate the **Counter.tsx** file with a simple counter and add the following contents:
+
+```ts
+import * as React from 'react';
+import Button from '@enact/moonstone/Button';
+
+export default class Counter extends React.Component {
```

Comment:
We have two `default` exports, this could just be `const class Counter...` instead.